src/index.ts

```diff
@@ -58,28 +60,23 @@ class DXScannerCommand extends Command {
     }
 
     let scanResult: ScanResult;
-    try {
-      scanResult = await scanner.scan();
-    } catch (error) {
-      if (error instanceof ServiceError && error.code === ErrorCode.AUTHORIZATION_ERROR) {
-        if (ScanningStrategyDetectorUtils.isGitHubPath(scanPath)) {
-          authorization = await cli.prompt('Insert your GitHub personal access token. https://github.com/settings/tokens\n', {
-            type: 'hide',
-          });
-        } else if (ScanningStrategyDetectorUtils.isBitbucketPath(scanPath)) {
-          authorization = await cli.prompt(
-            'Insert your Bitbucket credentials (in format "appPassword" or "username:appPasword"). https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html\n',
-            { type: 'hide' },
-          );
-        }
-
-        const container = createRootContainer({ uri: scanPath, auth: authorization, json, fail, recursive: flags.recursive });
-        const scanner = container.get(Scanner);
-
-        scanResult = await scanner.scan();
-      } else {
-        throw error;
+    scanResult = await scanner.scan();
+    if (scanResult.needsAuth && !isTravis) {
```

Comment:
`&& !isTravis` - why is the condition for travis here?

Reply:
Because otherwise the auth token prompt will appear also during Travis CI build (if the API limit is exhausted) and will stall the build.

Reply:
I guess it's not the best approach. If the user uses any other CI tool (e.g. Gitlab) so the PR will stall too. 

Q:
- Is there any error message if the scan runs in Travis (or any other tool) and the API limit is exhausted?
- Could we change the condition to `&& !flags.ci`?